FrameworkSystem/private/logging/Logger.py

```diff
@@ -250,36 +251,38 @@ def _processMessage( self, messageObject ):
       self._backendsDict[ backend ].doMessage( messageObject )
 
   def __getExceptionString( self, lException = False, lExcInfo = False ):
+    """
+    Return a formated string with exception and traceback information
+    If lExcInfo is present: full traceback
+    Elif lException is present: only last call traceback
+    Else: no traceback
+    """
+    if lExcInfo:
+      lException = False
     if lException:
+      # This is useless but makes pylint happy
```

Comment:
I think that setting lException to None as argument default value should make pylint happy as well